flyteplugins/go/tasks/plugins/k8s/spark/spark.go

```diff
@@ -322,39 +322,12 @@ func (sparkResourceHandler) BuildIdentityResource(ctx context.Context, taskCtx p
 }
 
 func getEventInfoForSpark(pluginContext k8s.PluginContext, sj *sparkOp.SparkApplication) (*pluginsCore.TaskInfo, error) {
-	state := sj.Status.AppState.State
-	isQueued := state == sparkOp.NewState ||
-		state == sparkOp.PendingSubmissionState ||
-		state == sparkOp.SubmittedState
-
 	sparkConfig := GetSparkConfig()
 	taskLogs := make([]*core.TaskLog, 0, 3)
 	taskExecID := pluginContext.TaskExecutionMetadata().GetTaskExecutionID()
 
-	if !isQueued {
```

Comment:
The diff view is unfortunate here. The only change is that `if !isQueued {` is removed.
The diff view suggests that "Driver logs" config was removed, the "User logs" config then turned into the "Drivers log" config, and so on.